assets/src/components/progressBar/style.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Progress bar styles.
+ */
+
+.muriel-progress-bar {
+	margin: 16px 0;
+	position: relative;
+	z-index: 1;
```

Comment:
This is kind of a fly-by, but given our aspirations to publish these components as a package for other consumers this might be a good moment to discuss `z-index` strategy. `z-index` normalization seems to be a major pain point for packaging Calypso components, since the mixin that handles  them works across the entirety of Calypso styles, which returns values which might be completely inappropriate in a different consumer App. Since this is (I believe) the first `z-index` in our component work, it would be great to strategize.

Reply:
[From here:](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context)
> The z-index values of its child stacking contexts only have meaning in [the] parent.

As far as I can tell, we should be OK in all situations here unless e.g. the wizard's stylesheet overrides the z-indexing of this component. In this specific situation, the z-index is required so that the gray part of the bar is placed under the pink part of the bar.

Can you provide an example of the issues you describe? I'm having a hard time coming up with a strategy without knowing what issues to avoid.